docker-compose.yml

```diff
@@ -5,7 +5,23 @@ services:
     restart: always
     volumes:
       - lodestar:/root/.local/share/lodestar
+      - /root/.local/share/lodestar/pyrmont:/root/.local/share/lodestar/pyrmont
     command: beacon --network ${NETWORK:-pyrmont}
+    expose:
+      - "9596"
+    network_mode: host
+
+  validator:
+    build: .
+    restart: always
+    volumes:
+      - lodestar:/root/.local/share/lodestar
+      # NOTE: mapping of pyrmont volume assumes that a validator has been manually imported or created here
+      - /root/.local/share/lodestar/pyrmont:/root/.local/share/lodestar/pyrmont
+    command: validator --network ${NETWORK:-pyrmont} --graffiti "lodestar wuz here"
```

Comment:
you should set beacon node url to be `http://lodestar:9596` then you won't need `network_mode: host` which would be recommended to avoid.

Reply:
:+1: For a configuration with network_mode: host now there's https://github.com/ChainSafe/lodestar/blob/master/docker/docker-compose-local.yml

The main docker-compose should use docker's DNS and only expose the necessary services

Reply:
@mpetrunic before i found out about `network_mode` i had already tried what you suggested, and i wasn't able to get the dockerized beacon node to accept requests from the dockerized validator

Reply:
@dapplion so assuming i keep the `network_mode: host`, then do you think i just add a beacon and validator service to that docker-compose-local.yml?

Reply:
You were probably missing this:
https://github.com/ChainSafe/lodestar/pull/2057#discussion_r579042163